internal/providers/pke/pkeworkflow/update_cluster.go

```diff
@@ -144,12 +171,15 @@ func UpdateClusterWorkflow(ctx workflow.Context, input UpdateClusterWorkflowInpu
 			ExternalBaseUrlInsecure:   input.PipelineExternalURLInsecure,
 			SSHKeyName:                "pke-ssh-" + input.ClusterName,
 		}
+		futures[np.Name] = workflow.ExecuteActivity(ctx, CreateWorkerPoolActivityName, createWorkerPoolActivityInput)
+	}
 
-		err := workflow.ExecuteActivity(ctx, CreateWorkerPoolActivityName, createWorkerPoolActivityInput).Get(ctx, nil)
-		if err != nil {
-			return err
+	caughtErrors = nil
```

Comment:
no need to assign `nil` because if we got here `caughtErrors` will still be `nil`